src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -18,7 +18,6 @@
 from pants.util.strutil import create_path_env_var
 
 
-# TODO: Support resources
```

Comment:
I'm not sure why this was here. We correctly inject resources like `Files` into the sources digest.